### PR TITLE
Add some docstring to get_coords_overlay

### PR DESCRIPTION
**Pull request by dstansby**
### Description
This adds a (incomplete) docstring to `get_coords_overlay`. Even though I didn't manage to track down doc for the `coord_meta` argument, this is still an improvement over no docstring 😄 

This pull request is to address ...

Fixes https://github.com/astropy/astropy/issues/8466

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Do the proposed changes actually accomplish desired goals?
- [x] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [x] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label. Codestyle issues can be fixed by the [bot](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#pre-commit).
- [x] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [x] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [x] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [x] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

File: astropy/visualization/wcsaxes/core.py [hunk 1]
```diff
@@ -596,6 +596,18 @@ def get_ylabel(self):
                 return coord.get_axislabel()
 
     def get_coords_overlay(self, frame, coord_meta=None):
+        """
```
Comment (pllim) [inline astropy/visualization/wcsaxes/core.py:599]: Also add a general description of the purpose of this method. Thanks!

File: astropy/visualization/wcsaxes/core.py [hunk 2]
```diff
@@ -596,6 +596,18 @@ def get_ylabel(self):
...
+        Parameters
+        ----------
+        frame : str, `~astropy.coordinates.BaseCoordinateFrame`
+            Frame to get overlay for. If a string must correspond to
+            one of the coordinate frames registered in the astropy
+            frame transform graph.
+
+        Returns
+        -------
+        overlay : `~astropy.visualization.wcsaxes.CoordinatesMap`
```
Comment (pllim) [inline astropy/visualization/wcsaxes/core.py:609]: Should have description... but don't just accept my suggestion. Fill in with actual info. Thanks!
```suggestion
        overlay : `~astropy.visualization.wcsaxes.CoordinatesMap`
            Overlay of blah blah blah.
```

**Comment by pllim** [comment]
Hopefully the maintainers pinged as reviewers would help you fill in the mysterious `coord_meta` here. 🙏

**Comment by dstansby** [comment]
I don't really have the cycles to add more info than is in this PR, so would be good to be merged as an improvement as is.

**Review by pllim — APPROVED** [review]
Thanks!

**Comment by pllim** [comment]
We only really need the RTD to be green here, so I am going to cancel irrelevant CI jobs.

**Pull request merged**